libp2p/Session.cpp

```diff
@@ -320,13 +326,11 @@ void Session::doRead()
     m_data.resize(h256::size);
     ba::async_read(m_socket->ref(), boost::asio::buffer(m_data, h256::size),
         [this, self](boost::system::error_code ec, std::size_t length) {
-            LOG_SCOPED_CONTEXT(m_logContext);
-
             if (!checkRead(h256::size, ec, length))
                 return;
             else if (!m_io->authAndDecryptHeader(bytesRef(m_data.data(), length)))
             {
-                cnetlog << "header decrypt failed";
+                LOG(m_netLoggerError) << "Header decrypt failed";
```

Comment:
This I guess can happen if we get corrupted or incorrectly encrypted header, I would leave it at DEBUG level, it's not very interesting on default verbosity